node-notes-cli/app.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs');
+const json = require('./data.json');
+const action = process.argv[2];
+
+const read = JSON.stringify(json, null, 2);
+
+if (action === 'read') {
+  console.log(read);
```

Comment:
The `read` action should format the list of notes like they are in the example video:

---

![image](https://user-images.githubusercontent.com/16143771/159055572-b2940cdb-f706-429c-8763-7c2c2a1d3b27.png)

---

![image](https://user-images.githubusercontent.com/16143771/159055691-f8c17b7d-1634-45c2-a8a4-7c4a52f0b6e8.png)